jenkins.go

```diff
@@ -503,7 +512,7 @@ func CreateJenkins(base string, auth ...interface{}) *Jenkins {
 		base = base[:len(base)-1]
 	}
 	j.Server = base
-	j.Requester = &Requester{Base: base, SslVerify: false}
+	j.Requester = &Requester{Base: base, SslVerify: true}
```

Comment:
"At the same time, not find the place to set CACert," -  what do you mean by that?